src/main/java/org/quartzpowered/network/codec/PacketCodec.java

```diff
@@ -46,9 +46,12 @@
 import java.util.List;
 
 public class PacketCodec extends ByteToMessageCodec<Packet> {
-    @Inject private Logger logger;
-    @Inject private SessionManager sessionManager;
-    @Inject private FactoryRegistry factoryRegistry;
+    @Inject
+    private Logger logger;
+    @Inject
+    private SessionManager sessionManager;
+    @Inject
+    private FactoryRegistry factoryRegistry;
 
```

Comment:
These are supposed to be on one line, please allow one line annotations in your formatting options and update the PR.